docs/reference-guide/query-modules/implement-custom-query-modules/api/c-api.md

```diff
@@ -2207,7 +2207,7 @@ enum mgp_error mgp_graph_create_edge(
 )
 ```
 
-Add a new directed edge between the two vertices with the specified label.
+Add a new directed edge between the two vertices with the specified label. When the first parameter is projected graph, it will create new directed edge with specified label only if both vertices are part of projected graph.
```

Comment:
```suggestion
Add a new directed edge between the two vertices with the specified label. When the first parameter is a projected graph, it will create a new directed edge with the specified label only if both vertices are a part of the projected graph.
```